src/services/database/cascadeDelete.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-param-reassign */
+import schemas from '../../schemas';
+import relationshipFinder from '../utils/relationshipFinder';
+import patch from '../utils/patchRelationship';
+
+const promise = [];
```

Comment:
This name should be plural and maybe more explicit. A promise of what? Maybe something like `deletionPromises`?